src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: all $transitioin-smooth ease;
```

Comment:
Don't use `all` in transition, it will affect in bad way on website performance
Specify only those properties that will be changed